### PR TITLE
Fix aarch64 being detected as x86_64

### LINKED ISSUE #2: Script fails on 64bit arm
On my Pine64 running ubuntu the getmic.ro script downloads linux64 rather than linux-arm.
This seems to be due to it checking for *"64" and getting confused.
```bash
$ uname -m
aarch64
```

### PR DESCRIPTION
**Pull request by Calinou**
This closes #2.

### CODE REVIEW

**Comment by Calinou** [comment]
@benweissmann Can you take a look at this, please? :slightly_smiling_face:

File: index.sh [hunk 1]
```diff
@@ -54,29 +54,29 @@ platform=''
 machine=`uname -m`
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-  if [[ "$machine" == *"64" ]]; then
```
Comment (benweissmann) [inline index.sh:57]: I'm concerned about breaking existing setups that are returning something that matches `*"64"` but not `"x86_64"` -- Can we just move the `arm` branch to be the first `if` here, add the `aarch` check, and leave the `*"64"` checks alone?

**Review by benweissmann — CHANGES_REQUESTED** [review]
Sorry for the delayed response! The change looks good but I'm a bit concerned about this breaking for existing users, so I think we should consider a more minimal change.

File: index.sh (hunk 1 above)
Comment (Calinou) [reply index.sh:57]: Sounds good to me, I updated the PR to check for ARM first.

**Comment by benweissmann** [comment]
Looks great, thanks for the contribution!

**Review by benweissmann — APPROVED** [review]

**Pull request merged**